src/chrome/content/rules/Amnesty-International.xml

```diff
@@ -4,7 +4,8 @@
 
 	Other Amnesty International rulesets:
 
-		- Amnesty.fr.xml
+        - Amnesty.fr.xml
+        - Amnesty.nl.xml
```

Comment:
Ah, I don't know actually. Maybe it was by accident. Is it okay for me to rebase?